{"name": "e-wallet 'pull' button", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'https://qa.t.../personalArea' == 'https://qa.t....co.il/wallet'\n  - https://qa.trado.co.il/wallet\n  ?                        ^  ^ ^\n  + https://qa.trado.co.il/user/personalArea\n  ?                        ^^^^^^^^^^^  ^^ ^", "trace": "driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"e10a3fab-bf0d-46fa-9337-730a77a95b10\")>\n\n    @allure.epic('Test Personal Area')\n    @allure.id(38)\n    @allure.title(\"e-wallet 'pull' button\")\n    @allure.description('Using the \"Pull\" button will redirect to the appropriate page(I don\"t know which page because nothing in this site works)')\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_e_wallet_pull(driver):\n        personal = P.PersonalPage(driver)\n        login = L.LoginPage(driver)\n        with allure.step('Logging in'):\n            login.log_in('0000000000', False, False)\n        with allure.step('Navigating to Personal area'):\n            login.click_login_signup()\n        with allure.step('clicking e-wallet'):\n            personal.click_(personal.E_WALLET_BTN)\n        with allure.step('Asserting we reached the E_wallet page'):\n>           assert driver.current_url == 'https://qa.trado.co.il/wallet'\nE           AssertionError: assert 'https://qa.t.../personalArea' == 'https://qa.t....co.il/wallet'\nE             - https://qa.trado.co.il/wallet\nE             ?                        ^  ^ ^\nE             + https://qa.trado.co.il/user/personalArea\nE             ?                        ^^^^^^^^^^^  ^^ ^\n\nTest_personal_area.py:68: AssertionError"}, "description": "Using the \"Pull\" button will redirect to the appropriate page(I don\"t know which page because nothing in this site works)", "steps": [{"name": "Logging in", "status": "passed", "start": 1680022661780, "stop": 1680022664700}, {"name": "Navigating to Personal area", "status": "passed", "start": 1680022664700, "stop": 1680022664980}, {"name": "clicking e-wallet", "status": "passed", "start": 1680022664981, "stop": 1680022665250}, {"name": "Asserting we reached the E_wallet page", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'https://qa.t.../personalArea' == 'https://qa.t....co.il/wallet'\n  - https://qa.trado.co.il/wallet\n  ?                        ^  ^ ^\n  + https://qa.trado.co.il/user/personalArea\n  ?                        ^^^^^^^^^^^  ^^ ^\n", "trace": "  File \"C:\\Users\\liad\\Documents\\GitHub\\Trado-Testing-project\\Test_personal_area.py\", line 68, in test_e_wallet_pull\n    assert driver.current_url == 'https://qa.trado.co.il/wallet'\n"}, "start": 1680022665250, "stop": 1680022665253}], "start": 1680022661779, "stop": 1680022665254, "uuid": "6eb608f3-c197-4a59-99be-a9e203412b13", "historyId": "062bc1f8104ebd5ff88d33994b8545f8", "testCaseId": "74e8fa45a8a985522ba3c4ab9b0dfb84", "fullName": "Test_personal_area#test_e_wallet_pull", "labels": [{"name": "as_id", "value": 38}, {"name": "epic", "value": "Test Personal Area"}, {"name": "severity", "value": "normal"}, {"name": "suite", "value": "Test_personal_area"}, {"name": "host", "value": "TWEEJ5"}, {"name": "thread", "value": "18924-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_personal_area"}]}